internal/lib/checkpoint.go

```diff
@@ -26,6 +32,27 @@ type ContainerCheckpointRestoreOptions struct {
 	libpod.ContainerCheckpointOptions
 }
 
+func addCheckpointImageMetadata(importBuilder *buildah.Builder, ctr *oci.Container) error {
```

Comment:
nit: can this function be defined below where it's last called?

Reply:
sure, done